app/javascript/controllers/audio_breakpoints_controller.js

```diff
@@ -25,11 +25,11 @@ export default class extends Controller {
   initMarkers() {
     this.breakpointMarkers = [...Array(this.segmentCountValue - 1).keys()]
```

Comment:
In the next iteration, I think the UI should have "segmentation" as its I/O rather than "breakpoints".  Since that's a more precise format (you can trim the ends of the file).